sunpy/map/sources/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+def _test_private_date_setters(map_object):
```

Comment:
This seems like a place people will not look for this, can you put it in a helpers.py?

Reply:
Do we want people to look for this?

Reply:
people writing or editing these tests may need to look for it.

Reply:
Moved